compiler-core/src/language_server/completer.rs

```diff
@@ -586,30 +592,46 @@ where
     }
 
     // Gets the position of the line with the first import statement in the file.
-    fn first_import_line_in_module(&'a self) -> Position {
-        let import_location = self
-            .module
-            .ast
-            .definitions
-            .iter()
-            .find_map(get_import)
-            .map_or(0, |i| i.location.start);
+    fn first_import_line_in_module(&'a self) -> (Position, bool) {
+        let first_import = self.module.ast.definitions.iter().find_map(get_import);
+        let import_location = first_import.map_or(0, |i| i.location.start);
         let import_location = self.module_line_numbers.line_number(import_location);
-        Position::new(import_location - 1, 0)
+        (
+            Position::new(import_location - 1, 0),
+            first_import.is_some(),
+        )
+    }
+
+    // Returns true if an extra newline should be added after the import statement.
+    // * ``import_location`` - The position of the first import statement in the source code.
+    fn add_newline_after_import(&'a self, import_location: Position, has_imports: bool) -> bool {
```

Comment:
Create a new enum rather than using a bool please, as you can't tell what `true` and `false` mean without reading the whole function for what it does with them.

Reply:
I'm kind of new to rust, not sure what's the best way... But added an enum, does this work?